packages/protect/src/lib/protect.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ * Copyright © 2025 Ping Identity Corporation. All right reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ *
+ */
+
+import { ProtectConfig, Protect } from './protect.types.js';
+
+// Add Signals SDK namespace to the window object
+declare global {
+  interface Window {
+    _pingOneSignals: {
+      init: (initParams?: ProtectConfig) => Promise<void>;
+      getData: () => Promise<string>;
+      pauseBehavioralData: () => void;
+      resumeBehavioralData: () => void;
+    };
+  }
+}
+
+/**
+ * @function createProtect - returns a set of methods to interact with the PingOne Signals SDK
+ * @param {ProtectConfig} config - the configuration options for the PingOne Signals SDK
+ * @returns {Protect} - a set of methods
+ */
+export function createProtect(config: ProtectConfig): Protect {
+  const options = config;
```

Comment:
Is this assignment necessary? We could rename the above parameter to `options` if we prefer that?